flutter_frontend_server/test/to_string_test.dart

```diff
@@ -60,7 +60,8 @@ void main() {
     ]));
     final runResult = io.Process.runSync(dart, <String>[regularDill]);
     checkProcessResult(runResult);
-    var paintString = '"Paint.toString":"Paint(Color(0xffffffff))"';
+    var paintString =
+      '"Paint.toString":"Paint(Color(alpha: 1.0000, red: 1.0000, green: 1.0000, blue: 1.0000, colorSpace: ColorSpace.sRGB))"';
```

Comment:
typically I'd call `Color.toString()` here but I'm unable to import `dart:ui` here for some reason.

Reply:
This is a VM test, so that is why.

Reply:
But it's weird that I'm asserting behavior for `dart:ui`, so it's there, it's just not visible to me.

Reply:
Its basically a compiler test though